bib/cmd/bootc-image-builder/image.go

```diff
@@ -226,19 +228,30 @@ func manifestForISO(c *ManifestConfig, rng *rand.Rand) (*manifest.Manifest, erro
 		customizations = c.Config.Customizations
 	}
 
-	img.Users = users.UsersFromBP(customizations.GetUsers())
-	img.Groups = users.GroupsFromBP(customizations.GetGroups())
+	img.Kickstart = &kickstart.Options{
+		Path:          "/osbuild.ks",
+		Users:         users.UsersFromBP(customizations.GetUsers()),
+		Groups:        users.GroupsFromBP(customizations.GetGroups()),
+		NetworkOnBoot: true,
+		OSTree: &kickstart.OSTree{
+			OSName: "default",
+		},
+	}
 	if kopts := customizations.GetKernel(); kopts != nil && kopts.Append != "" {
-		img.KickstartKernelOptionsAppend = append(img.KickstartKernelOptionsAppend, kopts.Append)
+		img.Kickstart.KernelOptionsAppend = append(img.Kickstart.KernelOptionsAppend, kopts.Append)
 	}
-	img.KickstartNetworkOnBoot = true
+
 	// XXX: this should really be done by images, the consumer should not
 	// need to know these details. so once images is fixed drop it here
 	// again.
-	if len(img.Users) > 0 || len(img.Groups) > 0 {
+	if len(img.Kickstart.Users) > 0 || len(img.Kickstart.Groups) > 0 {
 		img.AdditionalAnacondaModules = append(img.AdditionalAnacondaModules, "org.fedoraproject.Anaconda.Modules.Users")
 	}
 
+	// use lorax-templates-rhel if the source distro is not Fedora with the exception of Fedora ELN
```

Comment:
Should this really assume that every non-fedora distro needs this package thats only needed by centos and fedora ELN?

Reply:
i have somehow misplaced the source for the "only needed by centos and fedora ELN" claim